ci_framework/roles/build_containers/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure containers working directory exists
+  file:
+    path: "{{ cifmw_build_containers_basedir }}"
+    state: directory
+
+- name: Ensure logs directory exists
+  file:
+    path: '{{ cifmw_build_containers_basedir }}/logs'
+    state: directory
+
+- name: Install tcib
+  include_tasks: install.yaml
```

Comment:
ditto - ansible.builtin.include_tasks - and, in fact, that should be `ansible.builtin.import_tasks` in order to leverage tags properly.